src/ipyradiant/visualization/vis_tool.py

```diff
@@ -0,0 +1,114 @@
+import os
+import textwrap
+from pathlib import Path
+from rdflib import URIRef, Literal, Graph, BNode
+import traitlets as T
+from rdflib.namespace import RDF
+from pandas import DataFrame
+import ipywidgets as W
+from ipycytoscape import MutableDict, CytoscapeWidget
+
+default_edge = {"selector": "edge", "css": {"line-color": "blue"}}
+
+default_node = {"selector": "node", "css": {"background-color": "grey"}}
+
+
+class RDFVisualization(W.GridBox):
+    graph = T.Instance(Graph, allow_none=True)
+    cyto_widget = T.Instance(CytoscapeWidget, allow_none=True)
+    nodes = T.List()
+    click_output = T.Instance(W.Output)
+    click_output_box = T.Instance(W.VBox)
+    selected_node = T.Unicode(allow_none=True)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.output_title = W.HTML("<h3>Output from Clicks</h3>")
+        self.layout = W.Layout(grid_template_columns="60% 40%", height="500px")
```

Comment:
a `self.layout` will magically come into existence, you can set directly to it. I'd only update these if they don't already exist.